frontend/src/sass/components/_breadcrumbs.scss

```diff
@@ -1,8 +1,34 @@
-.breadcrumbs {
+nav.breadcrumb {
```

Comment:
is this back populated to the intake module?

Reply:
nope - we will do that.